xcode/Subconscious/Shared/Components/Common/EntryList/EntryListView.swift

```diff
@@ -7,17 +7,33 @@
 
 import SwiftUI
 
+enum EntryNotification {
+    case requestDetail(EntryStub)
+    case like(Slashlink)
+    case unlike(Slashlink)
+    case delete(Slashlink)
+    case requestLinkDetail(EntryLink)
+    case quote(Slashlink)
+}
```

Comment:
Death to all `onLink`/`onQuote`/`onDelete` callbacks

Reply:
![yeeees-gimli](https://github.com/subconsciousnetwork/subconscious/assets/58421/f4af328d-ec6c-4cca-93ef-23a51c4387df)